src/main/resources/model/instance.json

```diff
@@ -127,6 +127,23 @@
         }
       }
     },
+    "dates": {
+      "type": "object",
+      "properties": {
+        "dateTypeId": {
+          "index": "source",
+          "showInResponse": [ "search"]
+        },
+        "date1": {
+          "index": "keyword_lowercase",
+          "showInResponse": [ "search" ]
+        },
+        "date2": {
+          "index": "keyword_lowercase",
+          "showInResponse": [ "search" ]
+        }
```

Comment:
This 2 also should be `source` as we shouldn't support search directly by them.

Reply:
updated